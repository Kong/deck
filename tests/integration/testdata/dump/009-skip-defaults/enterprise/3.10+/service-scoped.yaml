_format_version: "3.0"
services:
- connect_timeout: 60000
  enabled: true
  host: example.com
  name: service-1
  plugins:
  - config:
      allow: null
      always_use_authenticated_groups: true
      deny:
      - example
      hide_groups_header: false
      include_consumer_groups: false
    enabled: true
    name: acl
    protocols:
    - grpc
    - grpcs
    - http
    - https
  - config:
      allowed_content_types:
      - application/json
      body_schema: '[]'
      content_type_parameter_validation: true
      parameter_schema:
      - explode: true
        in: query
        name: param-test
        required: false
        schema: '{"type":"string"}'
        style: form
      verbose_response: true
      version: kong
    enabled: true
    name: request-validator
    protocols:
    - grpc
    - grpcs
    - http
    - https
  routes:
  - https_redirect_status_code: 426
    name: r1
    path_handling: v0
    paths:
    - /r1/api/v1
    preserve_host: false
    protocols:
    - http
    - https
    regex_priority: 0
    request_buffering: true
    response_buffering: true
    strip_path: true
    tags:
    - tag1
  tags:
  - tag1
  - tag2
  write_timeout: 60000
