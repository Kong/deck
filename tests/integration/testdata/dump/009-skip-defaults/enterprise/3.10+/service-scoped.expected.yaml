_format_version: "3.0"
services:
- host: example.com
  name: service-1
  plugins:
  - config:
      always_use_authenticated_groups: true
      deny:
      - example
    enabled: true
    name: acl
  - config:
      body_schema: '[]'
      parameter_schema:
      - explode: true
        in: query
        name: param-test
        required: false
        schema: '{"type":"string"}'
        style: form
      verbose_response: true
    enabled: true
    name: request-validator
  routes:
  - name: r1
    paths:
    - /r1/api/v1
    tags:
    - tag1
  tags:
  - tag1
  - tag2
